src/engine/engineState.js

```diff
@@ -895,7 +942,7 @@ export class EngineState extends EventEmitter {
   // A server has sent address history data, so update the caches:
   handleHistoryFetch (
     scriptHash: string,
-    stateHash: string,
+    addressState: any,
```

Comment:
What's the expected type here? I'm pretty sure we defined this earlier somewhere.

Reply:
It's actually not defined as a type.
The object is:
{
          subscribed: boolean,
          synced: boolean,

          hash: string | null,
          // Timestamp of the last hash change.
          // The server with the latest timestamp "owns" an address for the sake
          // of fetching utxos and txids:
          lastUpdate: number,

          fetchingUtxos: boolean,
          fetchingTxids: boolean,
          subscribing: boolean
        }
But there is not type for it, I will define it since it's in use anyway